README.md

```diff
@@ -1,27 +1,89 @@
+https://github.com/microsoft/typescript-analyze-trace/issues/29
+You can run `npx analyze-trace --help` to find out about other options including:
 
-For a sorted list of compilation hot-spots, run `npx analyze-trace traceDir`.
-Pass `--help` to learn more about configuration options.
-For best results, run the analyzer on a machine where the paths in the trace file(s) resolve correctly.
+Option                    | Default | Description
+--------------------------|---------|-------------------------------------------------------------------------------
+`--skipMillis [number]`   | `100`   | Suppress events that take less than the specified number of milliseocnds
+`--color [boolean]`       | `true`  | Color the output to make it easier to read.
+`--expandTypes [boolean]` | `true`  | Expand the names of types when printing them.
+`--json [boolean]`        | `false` | *Experimental and unstable*: Produce JSON output for programmatic consumption.
 
-For a simplified view of a types file (useful when investigating an individual trace), run `npx simplify-trace-types traceDir\types.json output_path`.
+For a simplified view of a `types.json` file (useful when investigating an individual trace), you can run `npx simplify-trace-types traceDir/types.json output.txt`.
 Note that the resulting file is for human consumption and should not be passed to the analyzer (i.e. don't clobber the original).
 
-To pretty-print individual types from a types file (faster than processing the entire file), run `npx print-trace-types traceDir\types.json id+`.
+### Interpreting Results
+
+The `analyze-trace` output will try to highlight the most expensive portions of a compilation that it was able to measure (a.k.a. "hot spots").
+Each hot spot may have a breakdown of other contributing hot spots.
+
+`analyze-trace` will also try to point out when multiple versions of the same npm package was loaded and type-checked.
+
+Output currently looks like the following:
+
+```
+Hot Spots
+├─ Check file /some/sample/project/node_modules/typescript/lib/lib.dom.d.ts (899ms)
+├─ Check file /some/sample/project/node_modules/@types/lodash/common/common.d.ts (530ms)
+│  └─ Compare types 50638 and 50640 (511ms)
+│     └─ Compare types 50643 and 50642 (511ms)
+│        └─ Compare types 50648 and 50644 (511ms)
+│           └─ Determine variance of type 50492 (511ms)
+│              └─ Compare types 50652 and 50651 (501ms)
+|                 └─ ...
+├─ Check file /some/sample/project/node_modules/@types/babel__traverse/index.d.ts (511ms)
+└─ Check file /some/sample/project/node_modules/@types/react/index.d.ts (507ms)
+```
+
+Each step here is annotated with check times (e.g. checking two types in lodash took over 500ms).
+
+Some common messages
+
+Message | Explanation
+--------|------------
+"Compare types 1234 and 5678" | TypeScript had to check whether non-trivial types with internal IDs `1234` and `5678` were related.
```

Comment:
Why "non-trivial"?

Reply:
Because I checked the code and I saw that we map this from calls to `structuredTypeRelatedTo` - so we're not trying to compare `number` to `string` in these cases. Maybe a user doesn't care about that?

Reply:
Personally, I don't think that distinction is interesting.  Limiting the output to non-trivial types is basically just fallout from trivial comparisons being fast - it's not important to the abstraction.